src/wm/wm.vala

```diff
@@ -1172,7 +1172,7 @@ public class BudgieWM : Meta.Plugin
         if (cur_tabs == null) {
             return;
         }
-        switch_switcher();
+        switch_switcher("backward");
```

Comment:
Would it not just be cheaper to use a boolean here? Instead of string comparisons